ansible/roles_ocp_workloads/ocp4_workload_minio/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+ocp4_workload_minio_project: gpte-minio
+ocp4_workload_minio_project_display: Minio Object Storage
+
+# Port for the LoadBalancer Service
+ocp4_workload_minio_port: 9000
+ocp4_workload_minio_storage_size: "25Gi"
+
+# Access Credentials for Minio Service
+# Should be overwritten via Global Variables !!!
+ocp4_workload_minio_access_key: "minio_access_key"
+ocp4_workload_minio_secret_key: "minio_secret_key"
+
+# Minio Container Image
+ocp4_workload_minio_image: docker.io/minio/minio
```

Comment:
We should host this in quay so we don't run into any docker hub pull restrictions.

Reply:
This training isn't that popular. But we could.

Reply:
Up to you!